### ISSUE TITLE
./install_ubuntu.sh ends with errors:

### ISSUE THREAD

**Issue by perskes** [unix121/i3wm-themer#43]
After letting the script generate example files I am presented with this:

```
[>] Located the config file
[>] Located the json file
[>] Replacing the colors in .Xresources
[x] Failed to locate your .Xresources file
[>] Replacing the colors in your i3 configuration file
[x] Failed to locate your i3 configuration file
[>] Replacing your Polybar configuration file
[x] Failed to locate your polybar configuration file
[>] Replacing wallpaper
[x] Failed to locate your nitrogen configuration file
[>] Refreshing i3 and xrdb and setting wallpaper
xrdb: No such file or directory
xrdb: can't open file '/home/root/.Xresources'
[>] Done!
```
Not sure if I can leave it like this, but the python script is nowhere to be found, and change.sh 003 does "something" but it's immediately reversed to the previous (default) I3 design.

I installed i3 gaps, polybar and nitrogen manually, yet I get these errors and wonder how to proceed. Thank you!

btw. using mint19 and latest i3 gaps

**Comment by unix121** [comment]
Hello sir,
as far as I can tell from that log, you have probably given wrong values to your [config](https://github.com/unix121/i3wm-themer/blob/master/src/defaults/config.yaml) file. If that's the case, you should add your full path of your configuration files there.

If that's not the case a bit more information would be great.

**Comment by unix121** [comment]
I'm closing this since it should be fixed by #117 .
